esmvalcore/preprocessor/_derive/sispeed.py

```diff
@@ -0,0 +1,62 @@
+"""Derivation of variable `sispeed`."""
+
+from iris import Constraint
+from iris.coords import DimCoord
+
+from ._baseclass import DerivedVariableBase
+
+
+class DerivedVariable(DerivedVariableBase):
+    """Derivation of variable `sispeed`."""
+
+    # Required variables
+    required = [
+        {'short_name': 'usi', },
+        {'short_name': 'vsi', }
+    ]
+
+    @staticmethod
+    def calculate(cubes):
+        """
+        Compute sispeed module from velocity components siu and siv.
+
+        Arguments
+        ----
+            cubes: cubelist containing velocity components.
+
+        Returns
+        -------
+            Cube containing sea ice speed.
+
+        """
+        siu = cubes.extract_strict(Constraint(name='sea_ice_x_velocity'))
+        siu.remove_coord('year')
+        siu.remove_coord('day_of_year')
+
+        siv = cubes.extract_strict(Constraint(name='sea_ice_y_velocity'))
+        siv.remove_coord('latitude')
+        siv.remove_coord('longitude')
+        siv.remove_coord('year')
+        siv.remove_coord('day_of_year')
+
+        # Models usually store siu and siv in slightly different points
+        if isinstance(siu.coord('latitude'), DimCoord):
+            siv.add_dim_coord(
+                siu.coord('latitude'), siu.coord_dims('latitude')
+            )
+            siv.add_dim_coord(
+                siu.coord('longitude'), siu.coord_dims('longitude')
+            )
+        else:
+            siv.add_aux_coord(
+                siu.coord('latitude'), siu.coord_dims('latitude')
+            )
+            siv.add_aux_coord(
+                siu.coord('longitude'), siu.coord_dims('longitude')
+            )
+
+        speed = (siu ** 2 + siv ** 2) ** 0.5
+        speed.short_name = 'sispeed'
```

Comment:
There is no need to do this here, this is always set by the main derive function:
https://github.com/ESMValGroup/ESMValCore/blob/125931cac0b2c386d21dc2baba65746323cefd3d/esmvalcore/preprocessor/_derive/__init__.py#L114-L117

Reply:
Removed